src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt

```diff
@@ -22,10 +22,13 @@ class CommandsController(@Autowired private val commandsService: CommandsService
         exchange: ServerWebExchange?
     ): Mono<ResponseEntity<RefundTransactionResponseDto>> {
         return refundTransactionRequestDto.flatMap {
+            val ipAddress = exchange?.request?.remoteAddress?.address?.hostAddress;
```

Comment:
I think that this will return the Ingress controller address here since api calls will not go directly to POD here but from ingress controller to POD.

In this case I would use the  [X-Forwarded-For](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) header for logging purpose instead

Reply:
Done! Thanks for the hint. :)